mandatory/3-playing-computer.js

```diff
@@ -7,36 +7,44 @@
   Answer the following questions:
 
   1. This program throws an error. Why? (If you can't find it, try executing it).
+  // Because b in console.log(b) is not defined, it is a parameter (local variable) in f1 and f2 functions, which means it cannot be reached globally, we can only access it inside the f1 and f2 functions
   2. Remove the line that throws the error.
+  // console.log(b) under console.log(a) commented out
   3. What is printed to the console?
   4. How many times is "f1" called?
+  // f1 function will be called three times: explained in for loop inside if 
   5. How many times is "f2" called?
+  // f2 function will be called two times: explained in for loop inside else 
   6. What value does the "a" parameter take in the first "f1" call?
+  // 8
```

Comment:
Excellent explanation Omer, We discuss about this exercise together.  Just a little feedback here on question 6 asking what is the value of a in the line
const f1 = function (a, b)   which you comment on line 34

Reply:
> Excellent explanation Omer, We discuss about this exercise together. Just a little feedback here on question 6 asking what is the value of a in the line
> const f1 = function (a, b) which you comment on line 34

Thanks for the feedback Mya :) I reviewed the code and revised it for a cleaner explanation